reader.go

```diff
@@ -443,15 +443,20 @@ func decodeLineOfMediaPlaylist(p *MediaPlaylist, wv *WV, state *decodingState, l
 				return fmt.Errorf("Byterange sub-range offset value parsing error: %s", err)
 			}
 		}
-	case !state.tagScte35 && strings.HasPrefix(line, "#EXT-SCTE35:"):
-		state.tagScte35 = true
+	case !state.tagSCTE35 && strings.HasPrefix(line, "#EXT-SCTE35:"):
+		state.tagSCTE35 = true
 		state.listType = MEDIA
-		scteData := strings.Split(line[12:], ",")
-		for _, sep := range scteData {
+		SCTEData := strings.Split(line[12:], ",")
+		state.scte = new(SCTE)
+		for _, sep := range SCTEData {
```

Comment:
I guess so, i will look up that function after some time and make use of it/ write something similar for our purpose